plutus-core/cost-model/data/builtinCostModelA.json

```diff
@@ -527,6 +527,19 @@
             "type": "constant_cost"
         }
     },
+    "ripemd_160": {
+        "cpu": {
+            "arguments": {
```

Comment:
You must have run the benchmarks to get these numbers.  Which machine did you run them on?

Reply:
@kwxm - unfortunately I did not benchmark this as I stated above:

> This draft contains an artificial costing model for the new builtin - I copied keccak_256 related values for now.

That is for sure missing part :-( Sorry for the confusion.